### PR TITLE
added the initial set of test for commands.js - tested /search and /s…

### PR DESCRIPTION
**Pull request by pradeepmurugesan**
…hortcuts

#### Summary
added the initial set of test for commands.js

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/12005

#### Related Pull Requests

- Has server changes - NO
- Has redux changes - NO
- Has mobile changes - NO

### CODE REVIEW

File: actions/command.test.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+jest.mock('utils/user_agent.jsx');
+jest.mock('actions/global_actions.jsx');
+
+describe('executeCommand', () => {
+    let store;
+    beforeEach(async () => {
+        store = await mockStore(initialState);
+    });
+
+    test('should fire the UPDATE_RHS_SEARCH_TERMS with the terms', async () => {
```
Comment (marianunez) [inline actions/command.test.js]:
```suggestion
    test('search should fire the UPDATE_RHS_SEARCH_TERMS with the terms', async () => {
```

File: actions/command.test.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            {type: 'SEARCH_POSTS_REQUEST', isGettingMore: false},
+        ]);
+    });
+
+    describe('shortcuts', () => {
+        UserAgent.isMobile = jest.fn();
+
+        beforeEach(() => {
+            UserAgent.isMobile.mockReset();
+        });
```
Comment (hmhealey) [inline actions/command.test.js]: I don't think these should be necessary. I think `jest.mock` automatically mocks `UserAgent.isMobile` and `mockReset` is called automatically with how we've configured Jest

File: actions/command.test.js (hunk 1 above)
Comment (hmhealey) [reply actions/command.test.js]: Alternatively, you could wrap the whole thing in a `describe` block just in case we add more tests to this in the future

**Comment by pradeepmurugesan** [comment]
> Thanks @pradeepmurugesan! Are you open to complete the other commands `leave`, `settings` and `collapse/leave` to complete the tests for this ticket?

@marianunez Yes.. I want to test a couple of actions in this PR. Will cover the other actions in separate PRs. Since it's a big file, it will be easy to review in case of multiple PRs.

**Comment by marianunez** [comment]
@pradeepmurugesan just a reminder that always try and avoid force pushing a PR that's actively being reviewed.
See https://developers.mattermost.com/blog/submitting-great-prs/#4-avoid-force-pushing

**Comment by marianunez** [comment]
Added new GH issue for the remainder of the tests here: https://github.com/mattermost/mattermost-server/issues/12849

**Pull request merged**